packages/top-nav/src/TopNavItem.ts

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    html,
+    CSSResultArray,
+    TemplateResult,
+    PropertyValues,
+    ifDefined,
+    property,
+    query,
+} from '@spectrum-web-components/base';
+import { Focusable, LikeAnchor } from '@spectrum-web-components/shared';
+
+import itemStyles from '@spectrum-web-components/tabs/src/tab.css.js';
+import topNavItemStyles from './top-nav-item.css.js';
+
+/**
+ * @slot icon - The icon that appears on the left of the label
+ */
+
+export class TopNavItem extends LikeAnchor(Focusable) {
+    public static get styles(): CSSResultArray {
+        return [itemStyles, topNavItemStyles];
+    }
+
+    @query('a')
+    private anchor!: HTMLAnchorElement;
+
+    @property({ type: Boolean, reflect: true })
+    public selected = false;
+
+    public value = '';
+
+    public get focusElement(): HTMLAnchorElement {
+        return this.anchor;
+    }
+
+    public click(): void {
+        this.anchor.click();
+    }
+
+    protected render(): TemplateResult {
+        return html`
+            <a
```

Comment:
Could potentially abstract into a directive or even plain vanilla function to share render logic across anchor-likes

Reply:
This is _sort of_ what goes on in https://github.com/adobe/spectrum-web-components/blob/394fc44d18f52d3d490ad30b049372acae05b70f/packages/shared/src/like-anchor.ts#L58-L80, but we're still working out the sharing kinks for elements like this one that need a few custom attributes. I've been leaning away from using this [spread directive](https://open-wc.org/developing/lit-helpers.html#regular-spread) because I worry about it feeling a bit like a hack, but with [`lit-html@2`](https://github.com/Polymer/lit-html/issues/1182) thinking out the same it may be time to lean in. For this initial release, I'm gonna push off any decisions here, however.